packages/venia-concept/src/ServiceWorker/Utilities/catalogCacheHandler.js

```diff
@@ -0,0 +1,16 @@
+import { thirtyDays, catalogCacheName } from '../defaults';
+
+export const createCatalogCacheHandler = function() {
```

Comment:
I had to make this a function to accommodate the fact that the workbox object was loading after the JS Engine has parsed through these lines of code.